semgrep/semgrep/cli.py

```diff
@@ -418,6 +408,12 @@ def cli() -> None:
         help=argparse.SUPPRESS,
         # help="Legacy pattern recommendation functionality for use in semgrep-app playground",
     )
+    config.add_argument(
+        "--dangerously-allow-arbitrary-code-execution-from-rules",
```

Comment:
What now happens if you run with this flag?

Reply:
I wonder if we should output some kind of message warning users that they are using a deprecated flag.

Reply:
I agree with Emma here. I guess we don't want to keep this flag around forever, so we should warn that it is deprecated and will be removed in some future release.

Reply:
Done! I added a warning below :+1: